README.md

```diff
@@ -73,7 +73,7 @@ We need to create a webhook in the `Developers` section of Stripe. Pictured in t
 1. Enter your production deployment URL followed by `/api/webhooks` for the endpoint URL. (e.g. `https://your-deployment-url.vercel.app/api/webhooks`)
 1. Click `Select events` under the `Select events to listen to` heading.
 1. Click `Select all events` in the `Select events to send` section.
-1. Copy `Signing secret` as we'll need that in the next step.
+1. Copy `Signing secret` as we'll need that in the next step (e.g `whsec_xxx`) (/!\ be careful no to copy the webook id we_xxxx)
```

Comment:
```suggestion
1. Copy `Signing secret` as we'll need that in the next step (e.g `whsec_xxx`) (/!\ be careful no to copy the webook id we_xxxx).
```

Reply:
Small one to keep it consistent with others in the list 👍

Great catch! Happy to merge after this change is accepted!